.nyc_output/7dd98128-1d4d-4399-9568-ddc792e318ec.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Well done @Eazybee, i think there is a new line needed here or would your rather .nyc_output can be added to .gitignore.

Reply:
I will add it to git ignore.  It must have skipped my eyes.